rendercv/data_models.py

```diff
@@ -903,11 +903,12 @@ def url(self) -> str:
                 "GitHub": "https://github.com/",
                 "GitLab": "https://gitlab.com/",
                 "Instagram": "https://instagram.com/",
-                "Orcid": "https://orcid.org/",
+                "ORCID": "https://orcid.org/",
                 "Twitter": "https://twitter.com/",
                 "StackOverflow": "https://stackoverflow.com/users/",
                 "ResearchGate": "https://researchgate.net/profile/",
                 "YouTube": "https://youtube.com/",
+                "Google Scholar": "https://scholar.google.com/citations?user=",
```

Comment:
Adding a new social network requires other changes as well. I will document this in the developer guide; for now, we can omit this line for this pull request.

Reply:
Yes, please.